parquet-column/src/main/java/org/apache/parquet/schema/PrimitiveComparator.java

```diff
@@ -276,4 +279,24 @@ public String toString() {
       return "BINARY_AS_SIGNED_INTEGER_COMPARATOR";
     }
   };
+
+  /**
+   * This comparator is for comparing two float16 values represented in 2 bytes binary.
+   */
+  static final PrimitiveComparator<Binary> BINARY_AS_FLOAT16_COMPARATOR = new BinaryComparator() {
```

Comment:
Impl looks good, but we should add a test for the comparator [here](https://github.com/apache/parquet-mr/blob/45a1ae2d3b4078d3b9e83919cacb03050c1b8c88/parquet-column/src/test/java/org/apache/parquet/schema/TestPrimitiveComparator.java)

Reply:
+1

Reply:
added a test for the comparator!